lib/models/rules.js

```diff
@@ -132,6 +134,23 @@ function putR2core(rules, callback) {
     myutils.requestHelperWOMetrics('put', {url: config.perseoCore.rulesURL, json: rulesAndContexts}, callback);
 }
 
+/**
+ * Add ruleName if necessary in rule text field
+ *
+ * @param rule The rule object
+ */
+function normalizeRuleName(rule) {
+
+    var newAs = '"' + rule.name + '" as ruleName';
+    // Add "name as ruleName" if not exist
+    if (rule.text && rule.text.indexOf(' as ruleName') === -1) {
```

Comment:
Documentation should be changed to explain that now the name is not mandatory and which happens if the name is ommited (i.e. an example of autogenerated name would be great).

Reply:
I have changed the documentation but I do not know if I have approached it in the right way, please check if the change seems right. b356343